Automattic/jetpack
# AI Assistant: Add paragraph inline extension

Fixes #37450
Depends on D149323-code

## Proposed changes:

* Adds the "Make shorter" action to the paragraph block (inline or transformative extension alike)
* Adds the ParagraphHandler class and markdown tweaks for paragraphs

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply D149323-code to your sandbox
* Sandbox the public API
* Enable the inline extensions
* Go to the block editor
* Check that the Paragraph inline extension works like the Heading extension
* Use on a paragraph that has a line break on it (instead of broken down into two paragraph blocks)
* Check that requests work with this paragraph as well.

Known issue: Auto-scroll does not work yet.
